cypress/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+    "extends": "../configs/tsconfig.composite.json",
+    "compilerOptions": {
```

Comment:
One minor thing:

Since this project has only TS files, we could add:

```ts
"allowJs": false
```

as a compiler option, to disallow adding JS files here and force using TS :slightly_smiling_face:

Reply:
Done